repos/system_upgrade/cloudlinux/actors/checkpanelmemory/libraries/checkpanelmemory.py

```diff
@@ -14,7 +14,7 @@
     NOPANEL_NAME: 1536 * 1024,  # 1.5 Gb
     UNKNOWN_NAME: 1536 * 1024,  # 1.5 Gb
     INTEGRATED_NAME: 1536 * 1024,  # 1.5 Gb
-    CPANEL_NAME: 2048 * 1024,  # 2 Gb
+    CPANEL_NAME: 1836 * 1024,  # 1.8 Gb
```

Comment:
Is there any reasoning behind this specific values?

Reply:
Yes - cPanel requires more RAM than a no-panel system, but actually not 2 Gb exactly. In fact, requiring 2 Gb would block the upgrade on some systems that actually would have no OOM problems.

I noted this on our testing VMs (if you set exactly 2 Gb of memory for a VM and try to upgrade, this'll block - seeing less than _exactly 2 Gb_ available.
The cPanel team also let me know that this value is larger than necessary.

Reply:
Why don't put 1536 as for nopanel? Is there any reason why value for cpanel should be greater then for nopanel?